### DISCUSSION TITLE
Is the "eQ3 INNOGY RWE SE iM871A" USB Stick supported?

### DISCUSSION THREAD

**Discussion by garv3** [wmbusmeters/wmbusmeters#1099] [Q&A]
Hi everyone,

I would like to know if the eQ3 INNOGY RWE SE iM871A USB stick is compatible with the project. Especially with the Home Assistant add on.

Thanks on advance!

**Comment by masterxa** [comment]
Yes, it is compatible. But you need to load the driver for the "original" im871a

**Reply by masterxa** [reply]
should be:

sudo modprobe cp210x
sudo sh -c 'echo 10c4 87ed > /sys/bus/usb-serial/drivers/cp210x/new_id'

Apparently it wont survive a reboot, so put it in a startup-script

**Reply by masterxa** [reply]
https://community.home-assistant.io/t/get-innogy-smarthome-empfangseinheit-im871a-usb-adapter-to-work/617179

**Reply by garv3** [reply]
Yeah, that's gonna be the problem.

**Comment by garv3** [comment]
Well, I am using Home Assistant (OS). So I am not sure how to do this.

**Reply by masterxa** [reply]
You could try installing the ssh-addon

**Comment by the-ress** [comment]
Apparently, you can re-flash the firmware so it behaves like the original iM871A: https://github.com/wmbusmeters/wmbusmeters/issues/1055#issuecomment-1765223516
